rktlet/runtime/pod_sandbox.go

```diff
@@ -0,0 +1,246 @@
+	var ip string
+	if rktStatus.state == "running" {
+		apiStatus = runtimeApi.PodSandBoxState_READY
+		ip = parseRktNetworkIp(rktStatus.network)
+	}
+
+	createdUnix := rktStatus.created.Unix()
+	return &runtimeApi.PodSandboxStatusResponse{
+		Status: &runtimeApi.PodSandboxStatus{
+			Id:          req.PodSandboxId,
+			Metadata:    nil, // TODO
+			State:       &apiStatus,
+			CreatedAt:   &createdUnix,
+			Network:     &runtimeApi.PodSandboxNetworkStatus{Ip: &ip},
+			Linux:       nil, // TODO
+			Labels:      nil, // TODO
+			Annotations: nil, // TODO
+		},
+	}, nil
+}
+
+type rktStatusResponse struct {
+	state   string
+	created time.Time
+	network string
+}
+
+func parseRktStatus(status []string) rktStatusResponse {
+	resp := rktStatusResponse{}
+
+	for _, line := range status {
+		parts := strings.SplitN(line, "=", 2)
+		if len(parts) != 2 {
+			glog.Warningf("malformed rkt status response: expected an '=', got: %v", line)
+			continue
+		}
+
+		switch parts[0] {
+		case "state":
+			resp.state = parts[1]
+		case "created":
+			time, err := time.Parse("2006-01-02 15:04:05.999 -0700 MST", parts[1])
+			if err != nil {
+				glog.Warningf("error parsing rkt started time: %v", err)
+			}
+			resp.created = time
+		case "networks":
+			resp.network = parts[1]
+		}
+	}
+
+	return resp
+}
+
+// parseRktNetworkIp parses the `rkt status` formatted network into an ip.
+// The ip of a network named rkt.kubernetes.io will be preferred, followed by
+// default, followed by the first one
+// The input might look something like 'default:ip4=172.16.28.27,foo:ip4=x.y.z.a'
+func parseRktNetworkIp(networks string) string {
+	podNetworks := strings.Split(networks, ",")
+
+	foundIp := "" // best ip we've found yet by the criterion above
+	for _, network := range podNetworks {
+		networkParts := strings.Split(network, ":")
+		if len(networkParts) < 2 {
+			glog.Warningf("malformed rkt network part; should have at least one ':': %v", network)
+			continue
+		}
+		networkName := networkParts[0]
+		for _, networkIp := range networkParts[1:] {
+			networkIpParts := strings.Split(networkIp, "=")
+			if len(networkIpParts) != 2 {
+				glog.Warningf("malformed rkt networkIp part; should have one '=': %v", networkIp)
+				continue
+			}
+			if networkIpParts[0] != "ip4" {
+				// k8s only supports ipv4
+				continue
+			}
+
+			if networkName == "rkt.kubernetes.io" {
+				// Always prefer this network if available. We're done if we find it
+				return networkIpParts[1]
+			}
+			if networkName == "default" {
+				// even if we already have a previous ip, prefer default over it. If it
+				// was rkt.k8s we already returned, so it must have been an arbitrary
+				// one
+				foundIp = networkIpParts[1]
+			}
+
+			if foundIp == "" {
+				// If nothing else has matched, we can use this one, but keep going to
+				// see if we find 'default' or 'rkt.k8s.io'
+				foundIp = networkIpParts[1]
+			}
+		}
+	}
+	return foundIp
+}
+
```

Comment:
Good suggestion, I am actually consider to make the output in a json format, something like https://github.com/coreos/rkt/pull/3194